packages/opencensus-instrumentation-grpc/package.json

```diff
@@ -67,7 +67,7 @@
   "dependencies": {
     "@opencensus/core": "^0.0.19",
     "@opencensus/propagation-binaryformat": "^0.0.19",
-    "grpc": "~1.21.0",
+    "grpc": "1.24.2",
```

Comment:
`^1.24.2` would be more flexible and IMO fit better due to the nature of the dependency (per the _Additional context_ section on #715). Did you intentionally choose an exact version match?

Reply:
done in https://github.com/census-instrumentation/opencensus-node/pull/750/commits/a78335a7c7a542eb8f368034a061c7f6ab3f4eb7